src/utils/featheryClient/index.ts

```diff
@@ -454,10 +454,10 @@ export default class FeatheryClient extends IntegrationClient {
    */
   async _debouncedSubmitCustom(override: boolean) {
     if (Object.keys(this.pendingCustomFieldUpdates).length === 0) {
```

Comment:
when the debounced function is called and there are no pending changes, no need to make a request